### ISSUE TITLE
Pahan Style Ships

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO-Backers-Ships#3]
@tsunamayo Rather than keep posting in the other unrelated github post, I made this one specifically for the Pahan style ships. Took your advise and changed the style a bit for this ship. Do you have a paint job you want used for a police ship?

![](https://cdn.discordapp.com/attachments/406664114249793536/703709546354573412/unknown.png)

**Comment by tsunamayo** [comment]
Oh indeed very cool larger version, it feels different but in the same style! For the police, I would say lets try between white + blue with red accent version, and a black plus blue with red accent (basically switch white to black).
We could use it as a standard faction also if needed, we will see!
Cheers

**Comment by ZachZent** [comment]
![](https://cdn.discordapp.com/attachments/406664114249793536/703858515592085545/unknown.png)

I tried the white/grey and blue version with red highlights. Honestly I don't like how it looks. I think for a police force the black, grey, and blue would be best. I even debate the addition of red as it seems a bit out of place.

**Comment by tsunamayo** [comment]
Yes I see, but honestly white is not that bad, issue with black is obviously space is black ;)
When I said accent red is was just small contrasting touch typically the thruster, all the stuff that glows, weapon, and maybe the windows as a trow-back to my beloved lego haha.
You can send me the bp so I can have a look at them in-game!
Cheers

**Comment by ZachZent** [comment]
Here are the Pahan small and medium. I may try to build a large 100m in the future. I changed the detailing to red, but kept the black. Even though it is in space, they are still quite visible due to the bright skybox. Plus ships can "target" other ships so it isn't like it is super stealthy. And the final point, it looks cooler. If you do try a white paint, please share and prove me wrong.

I removed the turrets on both, but kept the hard points on each. I won't presume to know the "meta" nor am I really involved with the weapons discussion so I will let you add them how you wish. I am also awful at making turrets. Same with the amount of engines and shields on the medium ship. Later on Ill add interiors and more detailing on the medium, but for now these will work for an EA release.

https://drive.google.com/file/d/1fs-7K1OJBIQJpetFXD29ONuyXlxodTro/view?usp=sharing
Github doesn't seem to want to add zip files so here is a link to the two ships.

**Comment by tsunamayo** [comment]
@ZachZent just to confirm if you had updated these since or not. Thanks

**Comment by tsunamayo** [comment]
especially the interior for the bigger one, if you dont thats not a big deal as the player is not really supposed to see the interior, but I have to admit it is a shame as your interior build is just fantastic, and your door mechanism is definitely trailer material (I somehow missed it)...

**Comment by ZachZent** [comment]
@tsunamayo I have not updated them yet as I sadly haven't had time

**Comment by tsunamayo** [comment]
okay no pb!
